src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -152,14 +155,29 @@ internal abstract partial class AbstractNavigateToSearchService
         }
 
         private static async Task<ImmutableArray<SearchResult>> ComputeSearchResultsAsync(
-            Project project, Document searchDocument,
+            Project project, ImmutableArray<Document> priorityDocuments, Document searchDocument, 
             PatternMatcher nameMatcher, PatternMatcher containerMatcherOpt,
             DeclaredSymbolInfoKindSet kinds,
             ArrayBuilder<PatternMatch> nameMatches, ArrayBuilder<PatternMatch> containerMatches,
             CancellationToken cancellationToken)
         {
             var result = ArrayBuilder<SearchResult>.GetInstance();
-            foreach (var document in project.Documents)
+
+            // Prioritize the active documents if we have any.
+            var highPriDocs = priorityDocuments.Where(d => project.ContainsDocument(d.Id))
+                                               .ToImmutableArray();
+
+            var highPriDocsSet = highPriDocs.ToSet();
```

Comment:
Is `highPriDocsSet ` used?

Reply:
whoops.  i intended to use it in computing lowPriDocs to ensure the algorithm was linear and not quadratic.  good catch.